components/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This is just so I can build the test page. Will be removed in the final merge in favour of the build system @developit is working on.